src/pages/workspace/accounting/qbo/export/QuickbooksCompanyCardExpenseAccountPage.tsx

```diff
@@ -23,6 +24,9 @@ function QuickbooksCompanyCardExpenseAccountPage({policy}: WithPolicyConnections
     const policyID = policy?.id ?? '';
     const {exportCompanyCardAccount, exportAccountPayable, autoCreateVendor, errorFields, pendingFields, exportCompanyCard} = policy?.connections?.quickbooksOnline?.config ?? {};
     const isVendorSelected = exportCompanyCard === CONST.QUICKBOOKS_EXPORT_COMPANY_CARD.VENDOR_BILL;
+    const showAccountSelection = Boolean(autoCreateVendor) || (!isVendorSelected && exportCompanyCard);
```

Comment:
```suggestion
    const showAccountSelection = Boolean(autoCreateVendor || (!isVendorSelected && exportCompanyCard));
```
This condition is used in render and it should be boolean otherwise react will try to render the string in `exportCompanyCard`